README.md

```diff
@@ -35,17 +35,28 @@ As the journal grows over time, replaying billions of commands can take a signif
 **Did we mention how simple Memstate is to use?**
 
 ```csharp
-    // host db engine using default settings - database will be saved to disk as "DemoDatabase.journal"  
-    var settings = new MemstateSettings { StreamName = "DemoDatabase" };
-    var db = await new EngineBuilder(settings).BuildAsync<LoyaltyDB>();
-
-    int id = 1;
-    await db.ExecuteAsync(new InitCustomerIfNotExist(id, 10));
-    var customer = await db.ExecuteAsync(new EarnPoints(id, 20));
-    Console.WriteLine($"your new balance is {customer.LoyaltyPoints} points."); 
-
-    var top10 = await db.ExecuteAsync(new Top10Customers());
-    top10.ToList().ForEach(Console.WriteLine);
+        [Test]
+        public async Task Most_compact_start_using_all_default_configurations()
+        {
+            var engine = await Engine.Start<LoyaltyDB>();
```

Comment:
Just a note here, not suggesting you change,  but  `Engine.For<LoyaltyDB>()` is similar to `Engine.Start<LoyaltyDB>` except that it returns the same engine instance on multiple calls.

Reply:
Nice, I will create a minimalist unit test that shows the difference and when to use each.